nep-3-seconds.mediawiki

```diff
@@ -0,0 +1,177 @@
+<NEP: xxx>
+  Title: Reduce Block Time and Gas Generation Rate
+  Author: Jimmy Liao <vvvincentvan@gmail.com>, Vitor Nazário Coelho <vncoelho@gmail.com>,  Fernando Díaz Toledano <shargon@gmail.com>, Roman Khimov <roman@nspcc.ru>, Christopher Schuchardt <chris.schuchardt@neo.events> 
+  Type: Standard
+  Status: Draft
+  Created: 2025-02-11
+  Replaces: N/A
+  Hardfork: Echidna
+
+==Abstract==
+
+This NEP proposes two significant changes to the Neo N3 network parameters:
+1. Reducing the block time from 15 seconds to 3 seconds
+2. Reducing the GAS generation rate per block from 5 GAS to 1 GAS
+
+==Motivation==
+
+The primary motivations for these changes are:
+1. Improve transaction throughput and user experience
+2. Maintain a stable GAS/NEO ratio in the ecosystem
+3. Better compete with other high-performance blockchain platforms
+
+==Specification==
+
+===Block Time Reduction===
+* Current value: 15 seconds
+* Proposed value: 3 seconds
+* Implementation: 
+  * Move block time configuration from dBFT plugin to Policy native contract
+  * Allow the council to configure block time through the Policy contract
+  * Consensus mechanism will read block time from the Policy contract
+
+===GAS Generation Rate Reduction===
+* Current value: 5 GAS per block
+* Proposed value: 1 GAS per block
+* Implementation: 
+  * Retain the existing governance model where council controls GasPerBlock
+  * Council will need to adjust GasPerBlock separately after block time changes
+  * Coordination between the two changes will be managed through governance
+
+===Policy Contract Enhancement===
+* New native method: `SetBlockTime(BigInteger milliseconds)`
```

Comment:
```suggestion
* New native method: `SetBlockGenTime(BigInteger milliseconds)`
```
I prefer a more clearer name since we already have a similar name of method `GetBlock` in `native Ledger`